### PR TITLE
Codechef HORSES solution in C

### PR DESCRIPTION
**Pull request by ritish099**
**Fixes issue: #4665**

**Changes:**

added CodeChef Racing horses (HORSES) Solution in C

### CODE REVIEW

File: code/online_challenges/src/codechef/HORSES/HORSES.c [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    int t, i, j, k, n;
+    scanf("%d", & t);
+    for (i = 1; i <= t; i++) 
+    {
+        scanf("%d", & n);
+        int s[n];
+        for (j = 0; j < n; j++)
+            scanf("%d", & s[j]);
+        int min = abs(s[0] - s[1]);
+        for (k = 0; k < n; k++) {
```
Comment (arnavb) [inline code/online_challenges/src/codechef/HORSES/HORSES.c]: This variable should be called j as well and the one inside k. The declaration of k should be above the innerr loop. Also use pre-increment whenever possible.

File: code/online_challenges/src/codechef/HORSES/HORSES.c [hunk 2]
```diff
@@ -0,0 +1,26 @@
+#include <stdio.h>
+
```
Comment (arnavb) [inline code/online_challenges/src/codechef/HORSES/HORSES.c]:
```suggestion
```

File: code/online_challenges/src/codechef/HORSES/HORSES.c [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+#include<stdlib.h>
+
+int main(void) 
+{
+    int t, i, j, k, n;
+    scanf("%d", & t);
+    for (i = 1; i <= t; i++) 
+    {
+        scanf("%d", & n);
```
Comment (arnavb) [inline code/online_challenges/src/codechef/HORSES/HORSES.c]: Declare n above this line.

**Comment by arnavb** [comment]
@ritish099 You just have to do my last suggestion.

**Comment by ritish099** [comment]
What ? @arnavb 
i think u removed the blank line ! what is the last suggestion for me

**Comment by arnavb** [comment]
Declare n above line 11.

**Pull request merged**